docs/terraformoptions.adoc

```diff
@@ -1000,14 +1006,39 @@ Refer to {uri-topology}[topology] for more thorough examples.
 |Default
 
 |enable_calico
-|Whether to install {uri-calico}[Calico] as {uri-calico-policy}[pod network policy].
+|Whether to install {uri-calico}[Calico] for pod {uri-calico-policy}[network policy] and/or {uri-calico-networking}[networking].
 |true/false
 |false
 
 |calico_version
 |Version of {uri-calico}[Calico] to install.
-|3.19
-|3.19
+|3.7-3.21, 3.24.1-3.24.5
+|3.24.5
+
+|calico_mode
+|{uri-calico}[Calico] installation {uri-calico-manifests}[mode].
+|canal, vxlan, flannel-migration, policy-only, ipip
+|policy-only
+
+|calico_mtu
+|{uri-calico-mtu}[Interface MTU] for {uri-calico}[Calico] device(s).
+|`8980` (IP-in-IP), `8950` (VXLAN), `0` (Auto)
+|0
```

Comment:
What does the value of 0 here equate to in https://projectcalico.docs.tigera.io/networking/mtu?

Reply:
It will leave it unset for Calico's default:
>By default, Calico will auto-detect the correct MTU for your cluster based on node configuration and enabled networking modes. This guide explains how you can override auto-detection of MTU by providing an explicit value if needed.